### DISCUSSION TITLE
Web UI Feedback using ISO installed 20221126

### DISCUSSION THREAD

**Discussion by djotaku** [rhinstaller/anaconda-webui#35] [General]
**First:** 
Something that I know is not new to the discussions around the installer - I think the user should be given the option to partition their disks and/or encrypt their disks. Most users most of the time will be fine with the defaults, but I am one of those who keeps his home partition on its own disk for simplicity's sake when doing a reinstall.

**Second:** 
The following feedback depends on the goal of a new Fedora installation. I like the return of a step-by-step installation rather than the current "look for options that need attention" installation method. However, there are a lot of options missing like time zone, setting the hostname, setting up custom networking like IPs and DNS, and deciding if you want a root user. 

So if the goal of the new installation flow is to get a user to an installed and working desktop as quickly as possible (and they can configure timezones, etc from a working desktop), then the current flow is fine. (I still think you should allow for partitioning/encryption options). 

**Other**
I note the current installer doesn't allow for package selection, so it's not bad that it's missing here. However, I wonder if there could be some benefit from giving users the option to install or not install certain packages at install time. Additionally, I think it could be pretty useful to allow the user to decide if they want sshd running from the beginning.

**Comment by jkonecny12** [comment]
Thanks for the feedback.

As mentioned in the blog post. This is not final version of the installer so more features are planned and will come.
